agreements/atiqsayyed.md

```diff
@@ -0,0 +1,85 @@
+stated in this section) patent license to make, have made, use, offer to sell, sell, import, and otherwise transfer the
+Work, where such license applies only to those patent claims licensable by You that are necessarily infringed by Your
+Contribution(s) alone or by combination of Your Contribution(s) with the Work to which such Contribution(s) were
+submitted. If any entity institutes patent litigation against You or any other entity (including a cross-claim or
+counterclaim in a lawsuit) alleging that your Contribution, or the Work to which you have contributed, constitutes
+direct or contributory patent infringement, then any patent licenses granted to that entity under this Agreement for
+that Contribution or Work shall terminate as of the date such litigation is filed.
+
+### 4.
+You represent that You are legally entitled to grant the above license.  If your employer(s) has rights to intellectual
+property that you create that includes your Contributions, you represent that you have received permission to make
+Contributions on behalf of that employer, that your employer has waived such rights for your Contributions to the
+`Company`, or that your employer has executed a separate Corporate CLA with the `Company`.
+
+### 5.
+You represent that each of Your Contributions is Your original creation (see section 7 for submissions on behalf of
+others). You represent that Your Contribution submissions include complete details of any third-party license or other
+restriction (including, but not limited to, related patents and trademarks) of which you are personally aware and which
+are associated with any part of Your Contributions.
+
+### 6.
+You are not expected to provide support for Your Contributions, except to the extent You desire to provide support. You
+may provide support for free, for a fee, or not at all. Unless required by applicable law or agreed to in writing, You
+provide Your Contributions on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied,
+including, without limitation, any warranties or conditions of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR
+A PARTICULAR PURPOSE.
+
+### 7.
+Should You wish to submit work that is not Your original creation, You may submit it to the `Company` separately from
+any Contribution, identifying the complete details of its source and of any license or other restriction (including, but
+not limited to, related patents, trademarks, and license agreements) of which you are personally aware, and
+conspicuously marking the work as "Submitted on behalf of a third-party: [named here]".
+
+### 8.
+You agree to notify the `Company` of any facts or circumstances of which you become aware that would make these
+representations inaccurate in any respect.
+
+
+### Please sign below
+
+Full-Name              : Atiq Sayyed
+Date                   : 010/27/2015
```

Comment:
Thanks for the PR. Please check the date. Would be best in this format : `23 Sept 2015`